src/main/resources/leonardo-example.conf

```diff
@@ -0,0 +1,113 @@
+# Example conifg file for leonardo.
+# Anything commented out here will have have defaults specified or is not necessary to run leo.
+# All values can be modified for your specific google environment, database, and secruity certificates
+
+
+# Google Cloud dataproc configuration
+dataproc {
```

Comment:
i reckon you want to explain what each of these keys mean, especially since users are *required* to fill them in

Reply:
maybe even give the mandatory ones obviously placeholder values, like `YOUR_PROJECT_HERE` for `leoGoogleProject`

Reply:
I was thinking about `leoGoogleProject`. Based on how it's used it's really "the project to create the init bucket in", which for us is Leo's Google project. Might be nice to rename it to something more specific (doesn't have to be this PR).